### PR TITLE
OYchain apply for networkID 126, thanks, details please see   EIP155-126.json

### PR DESCRIPTION
**Pull request by nfttoken**
OYchain apply for networkID 126, 
details information please see   EIP155-126.json, 

thanks very much

### CODE REVIEW

File: _data/chains/EIP155-126.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "name": "OYchain Testnet",
+    "chain": "OYchain",
+    "rpc": [https://rpc.cntop3.com],
+    "faucets": [http://faucet.cntop3.com],
```
Comment (ligi) [inline _data/chains/EIP155-126.json:5]:
```suggestion
    "rpc": ["https://rpc.cntop3.com"],
    "faucets": ["http://faucet.cntop3.com"],
```

**Review by ligi — CHANGES_REQUESTED** [review]
quotes are missing

**Review by ligi — CHANGES_REQUESTED** [review]
the explorer does not seem to be EIP3091 compatible - please fix or use "none"

**Comment by nfttoken** [comment]
> the explorer does not seem to be EIP3091 compatible - please fix or use "none"

thanks, change to  "none" now.

**Comment by nfttoken** [comment]
add "icon" today, and could you approve my PR, or what shoud I do?

**Comment by nfttoken** [comment]
> **kontinuum/test ** Expected — Waiting for status to be reported
how about this "kontinuum/test Expected — Waiting for status to be reported"?  google says that  to open a new PR in branch not master could fullfil the required.   should I  try it?

**Review by nfttoken — COMMENTED** [review]
explorer change to "none",and add icon

**Review by ligi — CHANGES_REQUESTED** [review]
the filename must be lowercase like the others

**Comment by nfttoken** [comment]
> the filename must be lowercase like the others

ok, thank  you .  
 I will close this PR, and open a new apply.  because this PR's build is failed.

**Comment by ligi** [comment]
you can also just change it here - it will build again once you change it

File: _data/chains/eip155-126.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "nativeCurrency": {
+      "name": "OYchain Token",
+      "symbol": "OY",
+      "decimals": 18
+    },
+    "infoURL": "https://www.cntop3.com",
+    "shortName": "oychain",
+    "chainId": 126,
+    "networkId": 126,
+    "slip44": 126
```
Comment (ligi) [inline _data/chains/eip155-126.json:20]:
```suggestion
    "slip44": 126,
```

**Review by nfttoken — COMMENTED** [review]
commit your suggest

**Pull request merged**